chemprop/utils.py

```diff
@@ -147,8 +159,13 @@ def load_scalers(path: str) -> Tuple[StandardScaler, StandardScaler]:
     features_scaler = StandardScaler(state['features_scaler']['means'],
                                      state['features_scaler']['stds'],
                                      replace_nan_token=0) if state['features_scaler'] is not None else None
-
-    return scaler, features_scaler
+    atom_descriptor_scaler = StandardScaler(state['atom_descriptor_scaler']['means'],
+                                            state['atom_descriptor_scaler']['stds'],
+                                            replace_nan_token=0) if state['atom_descriptor_scaler'] is not None else None
+    bond_descriptor_scaler = StandardScaler(state['bond_descriptor_scaler']['means'],
+                                            state['bond_descriptor_scaler']['stds'],
+                                            replace_nan_token=0) if state['bond_descriptor_scaler'] is not None else None
+    return scaler, features_scaler, atom_descriptor_scaler, bond_descriptor_scaler
```

Comment:
We need to check whether the keys `atom_descriptor_scaler` and `bond_descriptor_scaler` actually exist. Otherwise, it is an old checkpoint file, where the scalers were `None`. I think a simply check such as:

```
if "atom_descriptor_scaler" in state.keys():
    atom_descriptor_scaler = StandardScaler(..........
else:
    atom_descriptor_scaler = None
```
and an analogous check for bond_descriptors should be fine.

Currently, the code is not compatible with loading old checkpoint files.